index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="My First Portfolio">
     <title>Portfolio</title>
```

Comment:
Well done overall 💯 but in order to improve this, the page title, <title> in the <head> of the page should adequately and briefly describe the content of the page. For example:
Poor titles:

Microverse project | Your portfolio.
> Portfolio project


Better page titles:

> Mark's portfolio
> Mark full-stack web developer

Yours could be `Javed's Portfolio` for example 🤔

Reply:
Thank you for your feedback I already changed that.